scripts/vim-patch.sh

```diff
@@ -35,6 +37,27 @@ check_executable() {
   fi
 }
 
+clean_files() {
+  if [[ ${#CREATED_FILES[@]} -eq 0 ]]; then
+    return
+  fi
+
+  echo
+  echo "Created files:"
+  local file
+  for file in ${CREATED_FILES[@]}; do
+    echo "  • ${file}"
+  done
+
+  read -p "Delete these files (Y/n)? " -n 1 -r reply
+  echo
+  if [[ "${reply}" =~ ^[Yy]$ ]]; then
+    rm ${CREATED_FILES}
```

Comment:
:+1: can't hurt. Also, there was actually an error here; changed `${CREATED_FILES}` to `${CREATED_FILES[@]}`.